src/Rapidez.php

```diff
@@ -110,6 +112,26 @@ public function setStore(Store|array|callable|int|string $store): void
         config()->set('rapidez.root_category_id', $store['root_category_id']);
         config()->set('frontend.base_url', url('/'));
 
+        $magentoUrl = trim(
+            Config::getValue('web/secure/base_url', ConfigScopes::SCOPE_WEBSITE) ?? config()->get('rapidez.magento_url'),
+            '/'
+        );
+        if ($magentoUrl !== url('/')) {
```

Comment:
Any value in macro'ing the `UrlGenerator` helper for a function that compares urls like this?

Reply:
In most cases using a [Request::fullUrlIs()](https://github.com/laravel/framework/blob/28de931377aed9fc2935b62f4bbd4bedab3eca30/src/Illuminate/Http/Request.php#L235) or [Request::is()](https://github.com/laravel/framework/blob/28de931377aed9fc2935b62f4bbd4bedab3eca30/src/Illuminate/Http/Request.php#L211) is the way to go for comparing urls. 

Of course that only works with the current request and url, not one like `/` and the Magento url.
If you have any uses in mind i'll be happy to add it, if not i think it's use is probably too niche and we can add it later.